tests/integration/test_charm.py

```diff
@@ -246,6 +247,7 @@ async def test_upgrade(ops_test: OpsTest):
             assert "paddlejobs" in rule.resources
 
 
+@pytest.mark.skip("Due to https://github.com/canonical/training-operator/issues/170")
 @pytest.mark.abort_on_fail
 async def test_remove_without_resources(ops_test: OpsTest):
     """Test remove when no resources are present.
```

Comment:
why should we skip this test? I think testing successful remove is still valid

Reply:
just the docstring needs to be updated if we decide to not skip it

Reply:
There is another test case that removes the charm right before `test_upgrade`, if we leave this one, it will attempt to remove something that is not deployed. 

I think that in the long run these test cases have to be refactored.